deployments/ansible/roles/collector/tasks/win_fluentd_install.yml

```diff
@@ -0,0 +1,27 @@
+---
+# Set fluentd package sources for Windows
+- name: Set fluentd package sources for Windows
+  ansible.builtin.set_fact:
+    td_agent_base_url: https://packages.treasuredata.com
+    win_fluentd_msi: td-agent-{{td_agent_version}}-x64.msi
+
+# Download  fluentd for Windows
+- name: Download fluentd for windows
+  ansible.windows.win_get_url:
+    url: "{{td_agent_base_url}}/4/windows/{{win_fluentd_msi}}"
+    dest: "%TEMP%"
+  register: fluentd_msi
+
+# Install fluentd on Windows
+- name: Install Fluentd on Windows
+  ansible.windows.win_package:
+    path: "{{fluentd_msi.dest}}"
+    state: present
+
+# Push Custom conguration file for fluentd
```

Comment:
Please remove all the comments above the tasks. Task name supposed to be descriptive enough. No need to duplicate it